### ISSUE TITLE
Update check: update progress dialog is shown if channel has changed

### ISSUE THREAD

**Issue by josephsl** [josephsl/stationPlaylist#9]
Hi,

Observed in 7.x and in beta: if update channel changes and when update check is run, update progress dialog comes up and cannot be dismissed unless nVDA restarts.

Technical: make sure to check the above condition before launching the progress dialog.

Thanks.
